gui/frontend/src/dashboard/dashboard.coffee

```diff
@@ -161,6 +161,7 @@ angular.module "mindbenderApp.dashboard", [
     $scope.title = "Snapshot " + $routeParams.snapshotId
     $scope.loading = false
     $scope.hideLoader = true
+    $scope.tableTab = { active: false }
```

Comment:
Using a simple boolean variable doesn't work here. I believe this is because Bootstrap UI places the tabs directive in an isolated scope (https://github.com/angular-ui/bootstrap/blob/master/src/tabs/tabs.js#L93), and therefore using prototypal inheritance from an object is the only simple way to get around this to access the $scope set by the controller (see https://github.com/angular/angular.js/wiki/Understanding-Scopes).